From 148ae989e06f8bbcf6b2b65edfb636e6bea44687 Mon Sep 17 00:00:00 2001
From: Konstantin Makariev <kmakariev@motorola.com>
Date: Thu, 20 Aug 2015 15:49:23 -0500
Subject: [PATCH 033/959] IKKRNBSP-3337 moto-msm: adding MMI Synaptics config
 options

This adds configuration options needed to build MMI flavor of
Synaptics touch driver

Change-Id: I22944ce565306fb9a2dd57e4111fd163b5f91951
Signed-off-by: Konstantin Makariev <kmakariev@motorola.com>
Reviewed-on: http://gerrit.mot.com/778372
SLTApproved: Slta Waiver <sltawvr@motorola.com>
SME-Granted: SME Approvals Granted
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Reviewed-by: Jeffrey Carlyle <jeff.carlyle@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index b413fd65d08..d715d9bb0f6 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -15,3 +15,10 @@
 #
 
 CONFIG_MMI_DEVICE_DTBS=y
+# CONFIG_TOUCHSCREEN_SYNAPTICS_I2C_RMI4 is not set
+# CONFIG_TOUCHSCREEN_SYNAPTICS_DSX_RMI4_DEV is not set
+# CONFIG_TOUCHSCREEN_SYNAPTICS_DSX_FW_UPDATE is not set
+CONFIG_TOUCHSCREEN_SYNAPTICS_DSX_I2C=y
+CONFIG_TOUCHSCREEN_SYNAPTICS_DSX_RMI_DEV_MMI=y
+CONFIG_TOUCHSCREEN_SYNAPTICS_DSX_TEST_REPORTING=y
+CONFIG_TOUCHSCREEN_SYNAPTICS_DSX_FW_UPDATE_MMI=y
-- 
2.11.0

